inductiva/fluids/scenarios/fluid_block/fluid_block.py

```diff
@@ -186,14 +173,9 @@ def _(cls, simulator: DualSPHysics):  # pylint: disable=unused-argument
     return DUALSPHYSICS_CONFIG_FILENAME
 
 
-@FluidBlock.gen_aux_files.register
+@FluidBlock.create_input_files.register
 def _(self, simulator: DualSPHysics, input_dir):  # pylint: disable=unused-argument
-    pass
-
-
-@FluidBlock.gen_config.register
-def _(self, simulator: DualSPHysics, input_dir: str):  # pylint: disable=unused-argument
-    """Generates the configuration file for DualSPHysics."""
+    """Creates DualSPHysics simulation input files."""
 
     replace_params_in_template(
```

Comment:
Are you going to leave the update on the replace_params for a next PR?

I think you could do it here, while you are at it!

Reply:
Indeed it was, I viewed it wrong due to a line break.